templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,181 @@
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+		</div>
+	</section>
+</div>
+
+<div class="well well-sm provisional well-bold brdr-0 mrgn-bttm-0">
+	<div class="container">
+		<section>
+			<h2 class="h3 mrgn-tp-md">Contact us</h2>
+			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3">
+				<li><a href="#">Contact [Institution]</a></li>
+				<li><a href="#">[Top contact task 2]</a></li>
+				<li><a href="#">[Top contact task 3]</a></li>
+			</ul>
+		</section>
+		<div class="row">
+			<section class="col-md-8 pull-left mrgn-bttm-lg">
+				<h2 class="h3 mrgn-tp-md">About the [Institution]</h2>
+				<ul class="list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-2">
+					<li><a href="#">Mandate</a></li>
+					<li><a href="#">Organizational structure</a></li>
+					<li><a href="#">Transparency</a></li>
+					<li><a href="#">Job opportunities</a></li>
+					<li><a href="#">Reports</a></li>
+					<li><a href="#">Compliance</a></li>
+					<li><a href="#">Enforcement notifications</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li>More: <a href="#">About the [Institution]</a></li>
+				</ul>
+			</section>
+			<div class="col-xs-12 col-md-4 col-sm-5 pull-right">
+				<section class="lnkbx mrgn-bttm-lg">
+					<h2 class="h3">Minister</h2>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+				</section>
+				{%- include components/gc-follow-us.html inlineLayout=true -%}
```

Comment:
```suggestion
				{%- include components/gc-follow-us.html isInline=true -%}
```

Also, do not forget to adjust the include to the follow-us include on the **theme-topic** template too. (Note that the theme-topic should NOT display the inline version.)